user-contract-mon/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"log"
+
+	monitor "github.com/Eslam-Nawara/user-contract-mon/internal"
+	"github.com/NicoNex/echotron/v3"
+)
+
+func main() {
+	mon, err := monitor.NewMonitor("./.config")
+	if err != nil {
+		panic("failed to load configrations")
```

Comment:
you can log the error and exit using `os.exit(1)` for more user friendly approach